doc/tutorial/search/solution3.cpp

```diff
@@ -0,0 +1,26 @@
+#include <seqan3/io/stream/debug_stream.hpp>
+#include <seqan3/search/algorithm/search.hpp>
+#include <seqan3/std/view/subrange.hpp>
+
+using namespace seqan3;
+using namespace seqan3::search_cfg;
+
+int main()
+{
+    dna4_vector text{"CGCTGTCTGAAGGATGAGTGTCAGCCAGTGTAACCCGATGAGCTACCCAGTAGTCGAACTGGGCCAGACAACCCGGCGCTAATGCACTCA"_dna4};
+    fm_index index{text};
+
+    configuration const cfg = max_error{substitution{1}};
+
+    auto results = search(index, "GCT"_dna4, cfg);
+    std::sort(std::begin(results), std::begin(results));
+
+    debug_stream << "There are " << results.size() << " hits.\n";
+
+    using iter_t = decltype(std::begin(text));
+    for (auto && pos : results)
+    {
+        debug_stream << "At position " << pos << ": "
+                     << view::subrange<iter_t, iter_t>{std::begin(text) + pos, std::begin(text) + pos + 3} << '\n';
```

Comment:
you can use `view::slice` here, that would be easier to read:
```cpp
<< (text | view::slice(pos, pos + 3))<< '\n';
```
(you could also get rid of the iter_t decltype thing)

Reply:
Yes, it's definitely easier to read, but afaik `view::slice` is not going to be in the std?

Reply:
Hm, true. Drop and then take works but is less intuitive again. Can you try using `view::all(begin+pos, begin+pos+3)`?
According to the draft standard that should forward to subrange, but I never tried (it would still allow to get rid of the explicit template arguments).

Reply:
Mhm, unfortunately it doesn't work.
`view::all` takes only 1 argument. And it seems the forward to `view::subrange` would create an unsized subrange out of the given ForwardRange.

Reply:
But please use std::ranges::begin